.gitignore

```diff
@@ -1,62 +1,2 @@
-# Logs
```

Comment:
This is really stupid. It's like "we don't know what's inside, let ignore it all just in case". Be explicit...

Reply:
this is standard .gitignore for node...